feature-libs/product-configurator/rulebased/components/attribute/types/checkbox/configurator-attribute-checkbox.component.ts

```diff
@@ -26,9 +33,12 @@ export class ConfiguratorAttributeCheckBoxComponent
   ownerKey: string;
   expMode: boolean;
   attributeValue: Configurator.Value;
+  isDeltaRendering: boolean;
 
   attributeCheckBoxForm = new UntypedFormControl('');
 
+  protected configUtilsService = inject(ConfiguratorStorefrontUtilsService);
```

Comment:
@Uli-Tiger, could you. please, rename `configUtilsService` to `configuratorStorefrontUtilsService`?